src/get.js

```diff
@@ -162,6 +163,21 @@ export async function get({
               compressing.tgz
                 .uncompress(out, ffmpeg ? nwDir : cacheDir)
                 .then(() => resolve());
+            } else if (platform === "osx") {
```

Comment:
Could you add a comment above this saying something like `TODO: compressing package does not restore symlinks on MacOS`?